src/main/kotlin/no/nav/familie/ef/iverksett/infrastruktur/service/JsonSchemaService.kt

```diff
@@ -22,6 +22,7 @@ class JsonSchemaService(private val iverksettingRepository: IverksettingReposito
         val iverksettinger = iverksettingRepository.findAll()
 
         log.info("Starter oppdatering av ${iverksettinger.count()}")
+        iverksettinger.filterNot { it.data.vedtak.tilkjentYtelse?.startdato == null }
```

Comment:
Kommer sannsynligvis få 2 oppdateringer her for de 2 instanser om starter opp då den første ikke kommer være ferdig med oppdateringen når instans 2 starter

Reply:
Jeg har endret til restcontrolle nå.